genie-proto/src/main/proto/genie.proto

```diff
@@ -73,8 +76,29 @@ message JobMetadata {
     repeated string command_args = 14;
 }
 
+message Criterion {
+    repeated string tags = 1;
+    string id = 2;
+    string name = 3;
+    string status = 4;
+    string version = 5;
+}
+
+message ExecutionResourceCriteria {
+    repeated Criterion cluster_criteria = 1;
+    Criterion command_criterion = 2;
+    repeated string application_ids = 3;
```

Comment:
Should we name this explicitly `_override` for clarity?

Reply:
Don't really care one way or another internally for this API. It was set to this for consistency with V3. I think in the database they're called `applications_requested` cause it doesn't necessarily guarantee that they're used or even exist.

Reply:
I was just suggesting in terms of readability. Someone that sees this would have to trace through a lot of code to figure out what it does, `_overrides` could provide a more intuitive hint as to its purpose.